psig_matcher/experiments/run_experiment.py

```diff
@@ -120,11 +120,45 @@ def run_meta_markov_multivariant_analysis(parts: List[Part], part_dim: int, num_
         collision_rate = estimate_overlap_of_set_with_sample_signals(multivariant_parts, num_samples, meta_pdf_ci, part_pdf_ci, confidence_bound)
         
         collisions.append(collision_rate)
+        log_metric("monte_carlo_collision_rate", collision_rate)
+
         lower, upper = compute_normal_ci(collisions, 0.95)
         confidence_ranges.append(upper - lower)
+        log_metric("monte_carlo_confidence_interval", upper - lower)
+
         # print(f"Estimated collision rate from sample distributiion has range: {upper - lower}")
         
         if len(confidence_ranges) > 100 and np.mean(confidence_ranges[-10:]) >= np.mean(confidence_ranges[-100:]):
+
+            # final approximated collision rate, and its associated CI
+            log_metric("monte_carlo_final_collision_rate", collision_rate)
```

Comment:
These would be redundant. ML Flow will store a series of metrics when logged multiple times, so we can just look at the last value in the series